pkg/tbtc/chain_test.go

```diff
@@ -59,6 +59,12 @@ type localChain struct {
 	depositSweepProposalValidationsMutex sync.Mutex
 	depositSweepProposalValidations      map[[32]byte]bool
 
+	pendingRedemptionRequestsMutex sync.Mutex
+	pendingRedemptionRequests      map[[32]byte]*RedemptionRequest
+
+	redemptionProposalValidationsMutex sync.Mutex
```

Comment:
Non-blocking but I think we should start using a single mutex per chain test class. This is escalating with every PR.

Reply:
Sure, we can do it in a follow-up PR.